diff --git a/cmake/WindowsSupport.cmake b/cmake/WindowsSupport.cmake
index 93c13094..d51a3b1a 100644
--- a/cmake/WindowsSupport.cmake
+++ b/cmake/WindowsSupport.cmake
@@ -30,9 +30,13 @@ endif()
 
 # Win32 file I/O
 set(win32_io_default OFF)
-if(WIN32)
-    set(win32_io_default ON)
-endif()
+# It seems that a recent patch in tifffile 4.1 may have broken
+# binary compatibility on windows.
+# https://gitlab.com/libtiff/libtiff/issues/173
+# https://github.com/python-pillow/Pillow/issues/4237
+#if(WIN32)
+#    set(win32_io_default ON)
+#endif()
 
 option(win32-io "Use Win32 I/O" ${win32_io_default})
 
